static/js/home.js

```diff
@@ -58,8 +58,13 @@ function SetDrawingEvent(drawingManager, map) {
       lastShape.setMap(null);
     }
 
+    drawingManager.setDrawingMode(null);
```

Comment:
Because the drawing only effects marker mode for now. If the mode is not 'accidents' this function will need to set drawing mode to null and return. And is the mode is 'accidents', it also need to set drawing mode to null.